state/watcher/hubwatcher.go

```diff
@@ -175,7 +182,10 @@ func newHubWatcher(hub HubSource, clock Clock, modelUUID string, logger Logger)
 func (w *HubWatcher) receiveEvent(topic string, data interface{}) {
 	switch topic {
 	case txnWatcherStarting:
-		// We don't do anything on a start.
+		if w.startingFunc != nil {
```

Comment:
I wonder if we want the passed in function, vs just always closing a channel that we always have that others can look to listen on. So something ilke:

```
// Started waits until the watcher has started and then returns.
// If the HubWatcher is already started, this returns immediately.
func HubWatcher Started() error {
  select {
    case <-tomb.Dying():
      return ErrDying ?(or 
    case <- s.started:
      return nil
}
```

I'm not sure we want the error return. We might just always return if we are dead or started, as it doesn't matter much and simplifies the code.